### ISSUE TITLE
CTRL+C not working when sh is set in rvm.sh

### ISSUE THREAD

**Issue by ykfq** [rvm/rvm#4505]
I found that `CTRL+C` does not working in terminal, and figured out that it was the `/etc/profile.d/rvm.sh` who cause the problem.

I changed it from `sh` to `bash`, and now its working properly.
`vim /etc/profile.d/rvm.sh`
```
if
  [ -n "${BASH_VERSION:-}" -o -n "${ZSH_VERSION:-}" ] &&
  test "`\command \ps -p $$ -o ucomm=`" != dash &&
 #test "`\command \ps -p $$ -o ucomm=`" != sh
  test "`\command \ps -p $$ -o ucomm=`" != bash
then
```

**Comment by awaltman** [comment]
This is possibly a duplicate of #4422.

**Comment by stbenjam** [comment]
It is, and the fix is 'rvm get head' to pull down the master branch of RVM.  The stable version of RVM has been broken for *months*. :-1:

**Comment by ykfq** [comment]
Closed due to duplicate to #4422 and had been fixed in  #4417

**Comment by pkuczynski** [comment]
We just released a new version 1.29.7 including this fix.

[Become a backer](https://opencollective.com/rvm#backer) and support us with a small donation to help us continue our activities. Thank you if you already one! 🙏
